src/microbe_stage/systems/StrainSystem.cs

```diff
@@ -0,0 +1,61 @@
+﻿namespace Systems
+{
+    using Components;
+    using DefaultEcs;
+    using DefaultEcs.System;
+    using DefaultEcs.Threading;
+    using Godot;
+    using World = DefaultEcs.World;
+
+    [With(typeof(CompoundStorage))]
```

Comment:
This system seems to only use StrainAffected and OrganelleContainer components now, so the attributes here need to be fixed.